examples/helia-remix/components/helia.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react'
+import { createHelia } from 'helia'
+
+export const initHelia = async () => {
+
+  const heliaNode = await createHelia()
+
+  const nodeId = heliaNode.libp2p.peerId.toString()
+  const nodeIsOnline = heliaNode.libp2p.status === 'started'
+
+  return { heliaNode, nodeId, nodeIsOnline }
+}
+
+const HeliaComponent: React.FC = () => {
+  const [id, setId] = useState<string | null>(null)
+  //@ts-expect-error : Type 'null' is not assignable to type 'HeliaNode'.
```

Comment:
i dont think we need this anymore?

Reply:
We actually need it because we are import it in the `_index.tsx` file